doc/release/multilocale.rst

```diff
@@ -17,15 +17,15 @@ This document outlines the steps to get started with multi-locale Chapel.
 
 #. Configure the Chapel runtime appropriately. For most configurations, set:
 
-  .. code-block:: bash
+   .. code-block:: bash
 
      export CHPL_COMM=gasnet
 
-  Then, consider changing several these variables:
+   Then, consider changing several these variables:
```

Comment:
I'm going to change it to "Then, consider changing these variables" while there. I don't think "several of" adds anything.